### PR TITLE
CORE-108 - Students Position in Waiting List

### PR DESCRIPTION
**Pull request by AxonC**

### CODE REVIEW

File: app/Listeners/Training/WaitingListEventSubscriber.php [hunk 1]
```diff
@@ -8,17 +8,22 @@ class WaitingListEventSubscriber
 {
     public function accountAdded($event)
     {
-        return Log::channel('stack')->info("Account {$event->account} ({$event->account->id}) was added to {$event->waitingList}");
+        return Log::channel('training')->info("Account {$event->account} ({$event->account->id}) was added to {$event->waitingList}");
     }
 
-    public function accountPromoted()
+    public function accountPromoted($event)
```
Comment (CalumTowers) [inline app/Listeners/Training/WaitingListEventSubscriber.php]: All of the logs need to tell us who performed the action too

File: app/Events/Training/AccountRemovedFromWaitingList.php [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    }
+
+    /**
+     * Get the channels the event should broadcast on.
+     *
+     * @return \Illuminate\Broadcasting\Channel|array
+     */
+    public function broadcastOn()
+    {
+        return new PrivateChannel('channel-name');
```
Comment (CalumTowers) [inline app/Events/Training/AccountRemovedFromWaitingList.php]: Now we're implementing logging properly, do we need to give this a channel name?

Comment (AxonC) [reply app/Events/Training/AccountRemovedFromWaitingList.php]: This isn't for logging - broadcast is for if we were using Laravel Echo and real-time events

Comment (CalumTowers) [reply app/Events/Training/AccountRemovedFromWaitingList.php]: Oops...

**Pull request merged**